gramps/gen/const.py

```diff
@@ -104,10 +105,20 @@
     else:
         USER_DATA = os.path.join(USER_HOME, "AppData", "Roaming", "gramps")
     USER_CONFIG = USER_DATA
+    # Migrate data from AppData\Local to AppData\Roaming on Windows.
+    OLD_HOME = os.path.join(USER_HOME, "AppData", "Local", "gramps")
+    if os.path.exists(OLD_HOME) and not os.path.exists(USER_DATA):
+        shutil.move(OLD_HOME, USER_DATA)
```

Comment:
We should at least log a warning if both exist.

Reply:
OK. I've added a warning.